javascript/javascript3/week1/weatherApp.js

```diff
@@ -0,0 +1,171 @@
+// All Buttons from html
+
+const input = document.querySelector('input')
+const buttonWithCity = document.querySelector('#cityButton')
+const locationButton = document.getElementById('locationButton')
+const saveLocationButton = document.getElementById('saveButton')
+const deleteLocationButton = document.getElementById('deleteLocationButton')
+
+// All class or id's from html
+
+const city = document.querySelector('#cityText')
+const temprature = document.getElementById('tempratureText')
+const feelsLike = document.getElementById('feelsLike')
+const minMaxTemp = document.getElementById('tempMinMax')
+const pressure = document.getElementById('pressure')
+const img = document.querySelector('img')
+const wind = document.getElementById('windSpeed')
+const cloud = document.getElementById('clouds')
+const sunTimes = document.getElementById('sunRiseSet')
+const mapDiv = document.getElementById('mapLocation')
+const mapLocation = document.querySelector('iframe')
```

Comment:
I know it can be tempting to define all these html elements up here so they are available for all functions (so-called global variables), and I like your overall idea here. However, I would personally prefer to define them in the places you actually need them (in the display functions in this case). The risk of having a lot of global variables is that you can have different functions change each one of them, so in larger code bases it can be really hard to track down and debug where the value is actually changed.

Reply:
Thanks for your feedback. Firstly I was did with the information. But after that when I decide to use different fetch for latitude and longitude, I need to take this informations outside. I used them 2 different functions. I will change thats location. I will try to make it more understandable and easy to read. Thanks again 👍